azure_sdk_core/src/lib.rs

```diff
@@ -151,6 +151,21 @@ pub trait ClientRequestIdOption<'a> {
     }
 }
 
+pub trait AppendPositionSupport<'a> {
```

Comment:
Very nice! You should also publish both these traits to the `prelude` module so the users could find it more easily! :+1: